validation/container_validator_test.go

```diff
@@ -71,6 +71,23 @@ var _ = Describe("ContainerValidator", func() {
 		Expect(validator.Validate(container)).NotTo(Succeed())
 	})
 
+	It("should succeed if topsort used, instead of set ", func() {
```

Comment:
Ah ok good, I will revisit this topic again on the weekend. I'm quite busy at the moment at work.